data.json

```diff
@@ -148,6 +148,26 @@
       "emoji": "🛠",
       "tags": ["rust", "tools", "gpu"],
       "description": "Wrapper crate for SPIRV-Tools"
+    },
+    {
+      "name": "review-action",
+      "emoji": "🏷️",
+      "tags": ["github-actions", "tools"],
+      "description": "Github Action that adds and removes labels based on PR review state"
+    },
+    {
+      "name": "nfd2",
+      "hidden": "This is a temporary fork so we are not showing it on the website"
```

Comment:
Clever idea - please add a line to the readme about how `hidden` should be used!

Reply:
I was just about to write up docs on all the other properties, and there they are in the README! I am a dummy